11.3.1 release note (#1110)
rei/rei-cedar-docs: docs/release-notes/fall-2021/README.md

<br>

## 11.3.1 

See what's changed on <cdr-link style="vertical-align: baseline" href="https://github.com/rei/rei-cedar/commit/c06fbeb241b97e29cad35d7eceea8fd15b583cdb">GitHub <icon-brand-github /></cdr-link>

### New Features

#### Icon Update

> We have introduced a new experiences icon to our [icon library](../../icons/library/)

Reviewer: ```suggestion
We've introduced a new experiences icon to our [icon library](../../icons/library/)
```

- Experiences running <icon-experiences-running/>

### Fixes

#### Mobile tokens

> Mobile tokens have been updated to ensure parity between Figma and the published package.

Reviewer: ```suggestion
We've updated mobile tokens to ensure parity between Figma and the published package.
```
